test/tests/ver-1hb/ver-1hb.i

```diff
@@ -0,0 +1,140 @@
+# R = '${units 8.31446261815324 J/K/mol}' # from PhysicalConstants
+# T = '${units 303 K}'
+# N_a = '${units 6.02214076e23 at/mol}' # from PhysicalConstants
+Q = '${units 0.1 m^3/s }'
+V = '${units 1 m^3}'
+Q_by_V = '${fparse Q / V}'
+
+[Mesh]
+    type = GeneratedMesh
+    dim = 1
+[]
+
+[Variables]
+    [P1_T]
+        initial_condition = '${units 1.0 Pa}'
+    []
+    [P2_T]
+        initial_condition = '${units 0 Pa}'
+    []
+
+    [P1_D]
+        initial_condition = '${units 0 Pa}'
+    []
+    [P2_D]
+        initial_condition = '${units 1.0 Pa}'
```

Comment:
Maybe move the initial pressure values to the top of the input file.

Reply:
Done.